src/Service/Suggestions/PostalUnit.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Gietos\Dadata\Service\Suggestions;
+
+use Gietos\Dadata\AbstractService;
+
+class PostalUnit extends AbstractService
+{
+    protected function getBaseUri(): string
+    {
+        return 'https://suggestions.dadata.ru/suggestions/api/4_1/rs';
+    }
+
+    public function suggest(string $query)
+    {
+        $body = ['query' => $query];
+        $request = $this->apiRequestFactory->createRequest('POST', $this->getBaseUri() . '/suggest/postal_unit', $body);
+        $response = $this->httpClient->sendRequest($request);
+        return json_decode((string) $response->getBody(), true);
+    }
```

Comment:
Why all responses are just decoded into plain array? Our goal is to provide concrete response objects.

See https://github.com/kstkn/dadata/tree/master/src/Model/Response for examples

Reply:
Ok, sorry for bothering